README.rst

```diff
@@ -66,6 +66,19 @@ Features
 * Should work with Python 3.6 and newer. (Versions 2019.2 and older continue
   to support Python 2.7.)
 
+Configuration
+-------------
+
+Override default key bindings
```

Comment:
This should explain that what you're supplying here is *partial* documentation of the config file. You need to explain more, in complete sentences. Where does the config file live? How does one change it? When do changes take effect?

Reply:
I have added a link to faq to avoid duplication

Reply:
@inducer Pls check the changes